website/forms/php/helpers.md

```diff
@@ -0,0 +1,58 @@
+---
+id: helpers
+title: Helpers
+---
+
+Eightshift Forms offers static helpers that can be implemented in your project. The key difference between these helpers and filters is that you can use the former in your theme or plugin code regardless of the load cycle.
+
+### esFormsGetFormIdByName
+
+This function retrieves the custom unique name of a form as set in the settings when provided with the form ID. When applying specific settings to one or more forms using project filters, it is handy. Usually, forms are identified by their IDs, but if you switch the environment, the form ID may change. This helper helps you identify the form by its name instead of its ID.
+
+```php
+if (\function_exists('esFormsGetFormIdByName')) {
+  echo esFormsGetFormIdByName('<form-id>');
+}
+```
+
+### esFormsDecryptor
+
+Decrypt and encrypt methods encode variations set in the form settings and passed to the thank you page upon form submission. This method lets you decode and use the variation in your theme or plugin.
```

Comment:
```suggestion
This helper provides decrypt and encrypt methods for variations set in the Form settings, passed to the "Thank you" page after form submission.
```
(this one was a bit _što je pisac htio reći_, so feel free to ignore it if I've missed the point 😄)